test_common/capybara_selectors/table_row.rb

```diff
@@ -0,0 +1,6 @@
+Capybara.add_selector(:table_row) do
```

Comment:
This is doing more than what its name suggests.   I would use this for selecting a table row, but its actually selecting a table row with a td that contains some text ?

Reply:
Fixed in f08cb64

Good point :)